### ISSUE TITLE
Question about fine-tuning to get f1 82-83%

### ISSUE THREAD

**Issue by BrambleXu** [ShomyLiu/pytorch-pcnn#5]
@ShomyLiu First, thanks for your work :)

I ran your code successfully. And I tried fine-tuning but end with MACRO-averaged f1 80.89%. How do you get f1 82-83% by fine-tuning? Could you share your thoughts?

**Comment by ShomyLiu** [comment]
Hi,
end with  **0.89%**?   or typos?
Since running codes directly without fine-tuning, it can reach about 80%

**Comment by BrambleXu** [comment]
It is a typo😱
It should be 80.89%

**Comment by ShomyLiu** [comment]
Be sure that you run the `test.sh` to evaluate the metric.  
You can try tuning different part parameters and find the best performance.

**Comment by BrambleXu** [comment]
Yes, I tried different parameters by running `test.sh` and got 80.89% finally.  I am curious about what your config settings to get 82-83%.

**Comment by ShomyLiu** [comment]
OK  I'll check is there any issues recently.
